README.md

```diff
@@ -21,7 +21,7 @@ Our vision for Files is to be fully functional while keeping it simple and easy
 ## Why should I contribute?
 We can confidently assert Files is the platform's best file explorer project to bring your innovations to. We focus heavily on extensibility and code modularity and have no plans to stop doing so. When you contribute to this project, you're helping everyone by fixing reported bugs, adding new features, or correcting existing behavior. These changes are quickly included in the final product for all users to benefit from.
 
-We welcome discussions and contributions on our repository, however to help maintain a healthy community, please check out our [code of conduct](https://github.com/files-community/Files/blob/main/CODE_OF_CONDUCT.md).
+We welcome discussions and contributions on our repository, however to help maintain a healthy community, please read our [code of conduct](https://github.com/files-community/Files/blob/main/CODE_OF_CONDUCT.md).
```

Comment:
We welcome discussions and contributions **on** -> **to** our repository | If I'm correct?